open-media-match/src/OpenMediaMatch/tests/test_api.py

```diff
@@ -15,12 +15,29 @@ def test_banks_empty_index(client: FlaskClient):
 
 
 def test_banks_create(client: FlaskClient):
+    # Must not start with number
     post_response = client.post(
         "/c/banks",
-        json={"name": "MY_TEST_BANK"},
+        json={"name": "01_TEST_BANK"},
+    )
+    assert post_response.status_code == 500
```

Comment:
This should be a 400 (user error) not 500 (server error). 

We would have to figure out how to validate ahead of time to get the right error type.